BL_C: 0.004457817207900782
BL_M: 1.1235870590741672
BR_C: 0.015046815033138213
BR_M: 1.114361706319497
GL_C: 0.007357931764883934
GL_M: 1.142015281034874
GR_C: 0.005459802874899981
GR_M: 1.1079177946391123
RL_C: 0.01179858271337318
RL_M: 1.098415857844346
RR_C: 0.004286066694761006
RR_M: 1.1099632654034857
